app/controllers/discourse_assign/assign_controller.rb

```diff
@@ -122,6 +122,28 @@ def assigned
       render json: { topics: serialize_data(topics, AssignedTopicSerializer) }
     end
 
+    def group_members
```

Comment:
I think we need a couple of security checks here. 

`can_assign` is already handled by the `before_action` in the controller ✅ 

I think we need `guardian.ensure_can_see_group_members!(group)` as well, in case the group is private. Can you also add that check to the list of assigned topics?